driver/ppm_events_public.h

```diff
@@ -1608,20 +1608,19 @@ enum extra_event_prog_code
 	PPM_SC_X(EPOLL_WAIT_OLD, 379) \
 	PPM_SC_X(RSEQ, 380) \
 	PPM_SC_X(CREATE_MODULE, 381) \
-	PPM_SC_X(PIDFD_GETFD, 382) \
-	PPM_SC_X(SCHED_GETATTR, 383) \
-	PPM_SC_X(FACCESSAT2, 384) \
-	PPM_SC_X(_SYSCTL, 385) \
-	PPM_SC_X(QUERY_MODULE, 386) \
-	PPM_SC_X(GET_MEMPOLICY, 387) \
-	PPM_SC_X(SYNC_FILE_RANGE, 388) \
-	PPM_SC_X(PROCESS_MADVISE, 389) \
-	PPM_SC_X(MEMBARRIER, 390) \
-	PPM_SC_X(IOPL, 391) \
-	PPM_SC_X(CLOSE_RANGE, 392) \
-	PPM_SC_X(FANOTIFY_MARK, 393) \
-	PPM_SC_X(RECV, 394) \
-	PPM_SC_X(SEND, 395)
+	PPM_SC_X(SCHED_GETATTR, 382) \
```

Comment:
NOTE: this is a somewhat API break; but:

* it was added very recently
* it is a generic event

Worst thing that could happen is that a scap file generated from a 0.10.0 libs onwards, will have all the generic events >= 382 shifted by one (ie: reading them wrong).
I know it's a bit bad, but it's not too bad luckily.

/cc @gnosek to resume discussion.

Reply:
I'd avoid adding the complexity of an unused PPM_SC code like Grzeg proposed, since this is a no-op change basically (again, let me stress this out, it is a somewhat breaking change, but has a super low impact).

Reply:
yeah same here I would avoid creating another exception in our PPM_SC codes just to handle this minor change...

Reply:
We could even manage a 0.10.5 libs tag with this fix, i think it would be worth it!

Reply:
makes sense :+1:

Reply:
@FedeDP what's the complexity of having an unused PPM_SC? It wouldn't be associated with any events and should be completely ignored by everybody.

I'm not going to fight very hard over it but I believe we should make 382 unused and avoid the renumbering (recv, send and faccessat2 seem like fairly popular syscalls to me)

Reply:
Yep i see no issues in doing that @gnosek . 
It's just that i didn't like having the unused PPM_SC given the minimum impact, but it's ok either way;
I'll propose a new version of this pr and ping you back!

Reply:
I implemented a new version where the old value is left as unused.

Reply:
Reverted to the initial implementation due to added complexity.